packages/soundux/soundux.pacscript

```diff
@@ -20,5 +21,13 @@ build() {
 }
 
 install() {
-  sudo make install DESTDIR=/usr/src/pacstall/soundux
+  sudo make install DESTDIR="$pkgdir"
+  # install binary symlink
+  sudo mkdir -p "$pkgdir/usr/bin/"
+  sudo ln -sf /opt/soundux/soundux "$pkgdir/usr/bin/soundux"
```

Comment:
>  Seems like it doesn't like absolute symlinks. Is there another to create that symlink or should I remove it?

Why are you symlinking this? What is the reason?

Reply:
I explained this in the description. That it is in the PATH.